boards/microchip/atmarktechno/degu_evk/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Degu Evaluation Kit configuration
```

Comment:
This board should not be part of the PR. It seems to be from another manufacturer and it is out of scope.

Reply:
I did not add or modify these entries. What do you want me to do ?
As you said, this is out of scope, so I should not touch them.

Reply:
You are submitting the code so you need Signed-off-by all commits and you will be responsible to make all review adjusts then. in this case, you must drop all this atmarktechno board.